packages/react/menu/src/Menu.tsx

```diff
@@ -186,11 +188,9 @@ const MenuSubTrigger = React.forwardRef((props, forwardedRef) => {
         onMouseMove={composeEventHandlers(props.onMouseMove, () => {
           if (!props.disabled) context.onOpenChange(true);
         })}
-        onKeyDown={composeEventHandlers(props.onKeyDown, (event) => {
-          if (!props.disabled && SUB_OPEN_KEYS[context.dir].includes(event.key)) {
-            context.onKeyOpen();
-          }
-        })}
+        onSearchExclusiveKeyDown={(event) => {
+          if (SUB_OPEN_KEYS[context.dir].includes(event.key)) context.onKeyOpen();
```

Comment:
Curious why we don't need the `!props.disabled` check here anymore?

Reply:
I did wonder if this was explicit enough. Will leave as is for now.